tools/serverpod_cli/lib/src/generator/dart/library_generators/model_library_generator.dart

```diff
@@ -1064,6 +1065,15 @@ class SerializableModelLibraryGenerator {
         return refer(field.type.className, serverpodUrl(serverCode))
             .property('fromString')
             .call([CodeExpression(Code(defaultValue))]).code;
+      case DefaultValueAllowedType.duration:
+        Duration parsedDuration = parseDuration(defaultValue);
+        return refer(field.type.className).call([], {
+          'days': literalNum(parsedDuration.days),
+          'hours': literalNum(parsedDuration.hours),
+          'minutes': literalNum(parsedDuration.minutes),
+          'seconds': literalNum(parsedDuration.seconds),
+          'milliseconds': literalNum(parsedDuration.milliseconds),
+        }).code;
```

Comment:
Question: Are there any tests that validate that this produces the correct result when hours has a value higher than 24? If not, I think we should add tests that cover all of these conversions from the code generator perspective.

Reply:
These could probably be added to the code generator tests.

Reply:
Done.